### ISSUE TITLE
:spell silently fails / does nothing

### ISSUE THREAD

**Issue by tylercecil** [mawww/kakoune#3330]
I just installed kak, and was unable to get `:spell` to do anything.

I'm using
- Kakoune v2019.12.10
- @(#) International Ispell Version 3.1.20 (but really Aspell 0.60.8)

Running `:spell`/`:spell en` does nothing, and `*debug*` is empty. Am I doing something wrong? What should I be expecting to happen?

**Comment by lenormf** [comment]
Could you try with the latest stable version, `v2020.01.16`?

There is a [mechanism in place](https://github.com/mawww/kakoune/blob/master/rc/tools/spell.kak#L59) to report errors in the latest version of the script.

**Comment by lenormf** [comment]
Dupe of #3254.

**Comment by tylercecil** [comment]
Ah, sorry about being out of date. Unfortunately I'm still not getting any output, or change of any kind. I'm not immediately sure how I can debug this...

**Comment by lenormf** [comment]
Here's a patch that should help you, it's an actual bugfix that I will submit in a PR in the near future:

```diff
From 90ba4d0903bc3e1af3d3f84db303d123222897d8 Mon Sep 17 00:00:00 2001
From: Frank LENORMAND <lenormf@gmail.com>
Date: Sat, 1 Feb 2020 15:55:32 +0100
Subject: [PATCH] rc spell: Handle errors returned on the first line

The first line returned by `aspell` isn't always an identification
string, it can also be an error.

This commit prevents the first line from being ignored in any case,
and allows errors to be reported consistently.

Related to #3330
---
 rc/tools/spell.kak | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/rc/tools/spell.kak b/rc/tools/spell.kak
index 47ada22f..0eeaceb5 100644
--- a/rc/tools/spell.kak
+++ b/rc/tools/spell.kak
@@ -40,9 +40,11 @@ define-command -params ..1 -docstring %{
             sed 's/^/^/' "$kak_opt_spell_tmp_file" | eval "aspell --byte-offsets -a $options" 2>&1 | {
                 line_num=1
                 regions=$kak_timestamp
-                read line # drop the identification message
                 while read -r line; do
                     case "$line" in
+                        @\(\#\)*)
+                            # drop the identification message
+                        ;;
                         [\#\&]*)
                             if expr "$line" : '^&' >/dev/null; then
                                pos=$(printf %s\\n "$line" | cut -d ' ' -f 4 | sed 's/:$//')
-- 
2.24.1

```

As for why the error doesn't show up on screen (you have to check the debug buffer even with the above fix), it's not clear to me yet how it should be fixed, but it will also address #3254.

**Comment by tylercecil** [comment]
Yeah, this patch worked. Thanks a bunch for getting that figured out so fast :) 
For posterity, the error was pretty obvious:
`Error: No word list can be found for the language en_US`
